src/api/hooks/useDuplicateProjectSearch.ts

```diff
@@ -0,0 +1,35 @@
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { ProjectsSearchParams, searchForProjects } from '../cristinApi';
+
+export const useDuplicateProjectSearch = (title: string | undefined) => {
```

Comment:
må sikkert ta inn `identifier` her, på samme måte som for rsultatsøk?
For nå får man advarsel om at prosjektet finnes fra før når man endrer alle prosjekt, siden vi søker opp det samme prosjektet